packages/bpk-component-autosuggest/src/BpkAutosuggest.module.scss

```diff
@@ -153,4 +153,14 @@ $arrow-size: bpk-spacing-sm() * 3;
       @include bpk-caption;
     }
   }
+
+  &__section-title {
+    padding: bpk-spacing-lg() 0 $bpk-spacing-xs $bpk-spacing-md;
```

Comment:
I think `bpk-spacing-xs` is deprecated, also we use the spacing functions so that it uses the new grid, can you replace it with whatever is the equivalent of these 2 tokens?